ui/src/i18n/pl.json

```diff
@@ -50,28 +50,28 @@
+  "backup.message_backup_help_one": "Utwórz kopię zapasową całego środowiska Homebridge, który następnie będziesz mógł przywrócic po ponownej instalacji Homebridge, lub po zresetowaniu systemu.",
   "backup.message_backup_help_two": "Wtyczki niedostępne z poziomu repozytorium npm lub instalowane bezpośrednio z GitHub nie będą dodane do backupu.",
-  "backup.message_backup_restored": "Przywracanie backupu zakończone sukcesem",
-  "backup.message_backup_warning": "Backup zawiera ważne informacje i nie powinien być udostępniany nikomu.",
-  "backup.message_restore_failed": "Przywracanie backupu zakończone niepowodzeniem",
-  "backup.message_restore_help_one": "Przywrócenie backupu odtworzy całą konfiguracje utworzoną za pomocą Homebridge UI wraz z accessorimai, plikiem konfiguracyjnym i wszystkimi wtyczkami zainstalowanymi z repozytorium npm.",
-  "backup.message_restore_help_two": "Po przywróceniu backupu trzeba ponownie zalogować się do Homebridge UI z danymi użytkownika zawartymi w tej kopii.",
+  "backup.message_backup_restored": "Przywracanie kopii zapasowej zakończone sukcesem",
+  "backup.message_backup_warning": "Kopia zapasowa zawiera ważne informacje i nie powinien być udostępniany nikomu.",
+  "backup.message_restore_failed": "Przywracanie kopii zapasowej zakończone niepowodzeniem",
+  "backup.message_restore_help_one": "Przywrócenie kopii zapasowej odtworzy całą konfiguracje utworzoną za pomocą Homebridge UI wraz z accessorimai, plikiem konfiguracyjnym i wszystkimi wtyczkami zainstalowanymi z repozytorium npm.",
+  "backup.message_restore_help_two": "Po przywróceniu kopii zapasowej trzeba ponownie zalogować się do Homebridge UI z danymi użytkownika zawartymi w tej kopii.",
   "backup.message_restore_max_size": "Możesz przywrócić archiwum kopii zapasowej do rozmiaru: {{ maxBackupSizeText }}.",
-  "backup.message_restore_warning": "Przywrócenie backupu jest nieodwracalne i zastąpi całą obecną konfigurację Homebridge.",
-  "backup.message_scheduled_backup_none_yet": "Nie utworzono jeszcze planowych backupów. Następny automatyczny backup zostanie wykonany o godzinie {{ backupTime }}.",
-  "backup.message_scheduled_backup_time": "Pełny backup konfiguracji Homebridge jest tworzony automatycznie każdego dnia o godzinie {{ backupTime }} i trzymany przez {{ dayCount }} dni.",
-  "backup.title_backup": "Backup - utwórz",
-  "backup.title_restore": "Backup - przywróć",
+  "backup.message_restore_warning": "Przywrócenie kopii zapasowej jest nieodwracalne i zastąpi całą obecną konfigurację Homebridge.",
+  "backup.message_scheduled_backup_none_yet": "Nie utworzono jeszcze planowych kopii zapasowych. Następny automatyczny backup zostanie wykonany o godzinie {{ backupTime }}.",
+  "backup.message_scheduled_backup_time": "Pełna kopia zapasowa konfiguracji Homebridge jest tworzona automatycznie każdego dnia o godzinie {{ backupTime }} i trzymany przez {{ dayCount }} dni.",
+  "backup.title_backup": "Kopia zapasowa",
```

Comment:
"backup.title_backup" and "backup.title_backup_restore" - you use same title, that is not correct, because one is to CREATE backup and second is to RESTORE backup.